### PR TITLE
Making LYTBoolGenerator print as true/false instead of 1/0

### PR DESCRIPTION
**Pull request by plivesey**

### CODE REVIEW

**Comment by nicksnyder** [comment]
lgtm

File: LayoutTestTests/LayoutTestCaseTests/LYTLayoutFailingTestSnapshotRecorderTests.m [hunk 1]
```diff
@@ -201,7 +201,7 @@ - (void)testSaveImageOfCurrentViewAddsFailureDescriptionImageAndDataToIndexFile
     dataHash = @"1252068542";
 #endif
     NSString *actualHTML = [self indexHTMLFile];
-    NSString *expectedBodyHTML = [NSString stringWithFormat:@"<TR><TD>This is a test failure description</TD><TD><IMG src='testSaveImageOfCurrentViewAddsFailureDescriptionImageAndDataToIndexFile/Width-100.00_Height-100.00_Data-%@.png' alt='No Image'></TD><TD>{    key = value;}</TD></TR>", dataHash];
+    NSString *expectedBodyHTML = [NSString stringWithFormat:@"<TR><TD>This is a test failure description</TD><TD><IMG src='testSaveImageOfCurrentViewAddsFailureDescriptionImageAndDataToIndexFile/Width-100.00_Height-100.00_Data-%@.png' alt='No Image'></TD><TD>{  \"key\" : \"value\"}</TD></TR>", dataHash];
```
Comment (drumnkyle) [inline LayoutTestTests/LayoutTestCaseTests/LYTLayoutFailingTestSnapshotRecorderTests.m]: Why is there an extra space before key but none after value?

**Comment by drumnkyle** [comment]
LGTM other than tiny comment.

File: LayoutTestTests/LayoutTestCaseTests/LYTLayoutFailingTestSnapshotRecorderTests.m (hunk 1 above)
Comment (plivesey) [inline LayoutTestTests/LayoutTestCaseTests/LYTLayoutFailingTestSnapshotRecorderTests.m]: Honestly...I do not know...

Comment (plivesey) [inline LayoutTestTests/LayoutTestCaseTests/LYTLayoutFailingTestSnapshotRecorderTests.m]: All this gets squashed anyway when it renders

**Pull request merged**